mitxgraders/formulagrader/formulagrader.py

```diff
@@ -947,8 +947,8 @@ def schema_config(self):
             Required('user_functions', default={}): {Extra: is_callable},
             Required('tolerance', default='5%'): Any(PercentageString, NonNegative(Number)),
             Required('samples', default=1): 1,
-            Required('variables', default=[]): [],
-            Required('numbered_vars', default=[]): [],
+            Required('variables', default=[]): All(Length(max=0), []),
```

Comment:
I believe that the usual empty-list testing, `Required('variables', default=[]): []`, is a bit broken in voluptuous `0.11.5`. I added `Length(max=0)` as a workaround, but this also gives a more informative error message IMO.

See https://github.com/alecthomas/voluptuous/issues/397

Despite this bug, I think upgrading is worth it.